practice-two/src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactNode, memo } from 'react';
+
+// item
+import './index.css';
+
+// type
+import { ButtonVariant } from '@type/button';
+
+export interface CustomButtonProps {
+    className?: string;
+    iconLeft?: ReactNode;
+    iconRight?: ReactNode;
+    title: string;
+    variant?: ButtonVariant;
+    disabled?: boolean;
+    loading?: boolean;
+    buttonType?: 'button' | 'submit' | 'reset';
+    onClick: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const renderContent = (
```

Comment:
why need to move this function outside this component?

Reply:
with the function have more than 3 param please use const functionName = ({}) = ()